api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -46,6 +50,17 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        String accessLocationUuid = Context.getAdministrationService().getGlobalProperty("locationbasedaccess.locationAttributeUuid");
+        if (personService.getPersonAttributeTypeByName("accessControlLocation") == null) {
```

Comment:
What is that if statement doing?

Reply:
I think, No need :smile:  
It will not replace or recreate a new personAttribute if it exist in the system.